### DISCUSSION TITLE
Project roadmap and/or release schedule

### DISCUSSION THREAD

**Discussion by vit9696** [buildbot/buildbot#8001] [General]
Hello!

Currently buildbot 4.0.2 has several bugs making our experience not so convenient. While it is discussible how critical the issues are, I would like to know whether there is any schedule for 4.0.3 release already or not so that we can plan the update policy? One can always update to a git branch and/or stay on a stable version, which we of course do, but in terms of planning knowing the timing is helpful.

To make the question more general, I understand that this is an open-source project, but perhaps there is any kind of written release cadence/roadmap making release schedule more transparent in general for the end users? Not asking for something strict, but perhaps [GitHub Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects) could be of some use?

Thanks!

**Comment by p12tic** [comment]
Buildbot 4.0.3 has been released.

Overall I want to keep releases happening at least once two months, ideally once a month.

I'm not sure if written release schedule will be of any use, because when there are delays, they happen due to availability reasons and in such case a release schedule that does not match reality would only confuse.

Best way to accelerate a release in case an problem that severely breaks your workflow is to create a new issue on github. It's not always obvious to a maintainer what's the impact of an issue, so making the impact clear makes it easy to prioritize a new release.

**Reply by flavius** [reply]
> Buildbot 4.0.3 has been released.
> 
> Overall I want to keep releases happening at least once two months, ideally once a month.
> 
> I'm not sure if written release schedule will be of any use, because when there are delays, they happen due to availability reasons and in such case a release schedule that does not match reality would only confuse.
> 
> Best way to accelerate a release in case an problem that severely breaks your workflow is to create a new issue on github. It's not always obvious to a maintainer what's the impact of an issue, so making the impact clear makes it easy to prioritize a new release.

Any thoughts on a higher-level roadmap? I think that would be useful. Without dates, but maybe versions.

Even no versions, but an outline of plans might at least show the community the direction.

Currently I have a feeling that there's no direction.
